grpc/src/main/java/com/linecorp/armeria/server/grpc/ArmeriaServerCall.java

```diff
@@ -150,7 +163,7 @@ public void sendHeaders(Metadata unusedGrpcMetadata) {
 
         HttpHeaders headers = HttpHeaders.of(HttpStatus.OK);
 
-        headers.set(HttpHeaderNames.CONTENT_TYPE, GRPC_PROTO_CONTENT_TYPE);
+        headers.set(HttpHeaderNames.CONTENT_TYPE, serializationFormat.mediaType().toString());
```

Comment:
You can just use `HttpHeaders.setObject()` instead of calling `toString()` explicitly.